File: Q1/Prog52a-Multilingual/main.jl
print("donne-moi ton longueur.\n")

len = readline()
len = parse(Int64, len) 

print("donne-moi ton largeur.\n")

wid = readline()
wid = parse(Int64, wid) 

area = len * wid

perim = ((len * 2) + (wid * 2))

print("your area is: ", area, "\n")
print("your perimeter is: ", perim, "\n")

File: Q1/Prog52a-Multilingual/main.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println("gimmie yo length so i can be done with this project")
  var len int
  fmt.Scanln(&len)

  fmt.Println("alr now the width and we're done")
  var wid int
  fmt.Scanln(&wid)

  var area int = len * wid
  var perim int = ((len * 2) + (wid * 2))

  fmt.Println("\n\nlord please let this work first try\n\n")

  fmt.Println("your area is: ", area)
  fmt.Println("your perimeter is: ", perim)
  
}
